controllers/topics.controllers.js

```diff
@@ -17,4 +17,16 @@ exports.getArticleById = (req, res, next) => {
       res.status(200).send({ article });
     })
     .catch(next);
-};
\ No newline at end of file
+};
+
+exports.getAllArticles = (req, res, next) => {
+
+  fetchAllArticles()
+    .then((articles) => {
+      const newArticles = articles.map(x => Object.assign({}, x, {
+        comment_count: parseInt(x.comment_count) //Object.assign because could not use spread operator
+      }));
+      res.status(200).send(newArticles);
```

Comment:
We should be returning the articles array on a key of articles here, rather than directly on the body.

Reply:
resolved